salt/modules/napalm_probes.py

```diff
@@ -0,0 +1,294 @@
+# -*- coding: utf-8 -*-
+'''
+NAPALM Probes
+=============
+
+Manages RPM/SLA probes on the network device.
+
+:codeauthor: Mircea Ulinic <mircea@cloudflare.com> & Jerome Fleury <jf@cloudflare.com>
+:maturity:   new
+:depends:    napalm
+:platform:   linux
+
+Dependencies
+------------
+
+- :doc:`napalm proxy minion (salt.proxy.napalm) </ref/proxy/all/salt.proxy.napalm>`
+
+.. versionadded: Carbon
+'''
+
+from __future__ import absolute_import
+
+# Import python lib
+import logging
+log = logging.getLogger(__file__)
+
+
+try:
+    # will try to import NAPALM
+    # https://github.com/napalm-automation/napalm
+    # pylint: disable=W0611
+    from napalm import get_network_driver
+    # pylint: enable=W0611
+    HAS_NAPALM = True
+except ImportError:
+    HAS_NAPALM = False
+
+# ----------------------------------------------------------------------------------------------------------------------
+# module properties
+# ----------------------------------------------------------------------------------------------------------------------
+
+__virtualname__ = 'probes'
+__proxyenabled__ = ['napalm']
+# uses NAPALM-based proxy to interact with network devices
+
+# ----------------------------------------------------------------------------------------------------------------------
+# property functions
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+def __virtual__():
+
+    """
+    NAPALM library must be installed for this module to work.
+    Also, the key proxymodule must be set in the __opts___ dictionary.
+    """
+
+    if HAS_NAPALM and 'proxy' in __opts__:
+        return __virtualname__
+    else:
+        return (False, 'The mdoule NTP cannot be loaded: \
+                napalm or proxy could not be loaded.')
+
+# ----------------------------------------------------------------------------------------------------------------------
+# helper functions -- will not be exported
+# ----------------------------------------------------------------------------------------------------------------------
+
+# ----------------------------------------------------------------------------------------------------------------------
+# callable functions
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+def config():
+
+    """
```

Comment:
Can you change these to single quotes (`'''`), please? We like single instead of double quotes for doc strings, according to our [style guide](https://docs.saltstack.com/en/latest/topics/development/conventions/style.html#single-quotes).